src/App.tsx

```diff
@@ -1,17 +1,20 @@
-import { useState } from 'react'
-import logo from './logo.svg'
-import './App.css'
+import { useState } from "react";
```

Comment:
can we change the rule for single quotes for preatier

Reply:
Yes, good idea 👍